launcher/MangoHud.cpp

```diff
@@ -75,8 +75,19 @@ QString getLibraryString()
     }
 
     for (QString vkLayer : vkLayerList) {
-        QString filePath = FS::PathCombine(vkLayer, "MangoHud.json");
-        if (!QFile::exists(filePath))
+        // prefer to use architecture specific vulkan layers
+        QStringList manifestNames = { "MangoHud.x86_64.json", "MangoHud.aarch64.json", "MangoHud.json" };
```

Comment:
It would probably be best to get the current system architecture instead of hardcoding specific names.

One annoying thing with that is that Meson decided to use a completely different name for arm64, so that would need to be a specific edge case.

```cpp
QString currentArch = QSysInfo::currentCpuArchitecture();

// MangoHud relies on meson to provide architecture names and meson favors aarch64 over arm64
if (currentArch == "arm64") {
    currentArch = "aarch64";
}
 
QStringList manifestNames = { QString("MangoHud.%1.json").arg(currentArch), "MangoHud.json" };
```

Reply:
good idea!